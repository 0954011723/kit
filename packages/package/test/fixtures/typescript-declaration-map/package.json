{
	"name": "typescript-declaration-map",
	"private": true,
	"version": "1.0.0",
	"description": "standard typescript package with declarationMap:true",
	"type": "module"
}
