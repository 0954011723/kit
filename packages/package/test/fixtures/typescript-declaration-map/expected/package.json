{
	"name": "typescript-declaration-map",
	"private": true,
	"version": "1.0.0",
	"description": "standard typescript package with declarationMap:true",
	"type": "module",
	"exports": {
		"./package.json": "./package.json",
		"./Test.svelte": "./Test.svelte",
		".": "./index.js"
	},
	"svelte": "./index.js"
}